dotnet/Directory.Packages.props

```diff
@@ -3,12 +3,13 @@
     <!-- Enable central package management -->
     <!-- https://learn.microsoft.com/en-us/nuget/consume-packages/Central-Package-Management -->
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
```

Comment:
Who comes up with these things?

Reply:
Should have gone with `ShouldPinCentrallyManagedTransitivePackageVersions` IMO, way better.